mgnify_pipelines_toolkit/analysis/assembly/process_dbcan_result_euk.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# Copyright 2023-2025 EMBL - European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import argparse
+import fileinput
+import logging
+from pathlib import Path
+import re
+
+logging.basicConfig(level=logging.INFO)
+
+
+def main(hmm_file, overview_file, genome_gff, outfile, dbcan_version):
+
+    hmm_path = Path(hmm_file)
+    overview_path = Path(overview_file)
+
+    if not hmm_path.exists():
+        raise FileNotFoundError(f"Input hmm path does not exist: {hmm_file}")
+
+    if not overview_path.exists():
+        raise FileNotFoundError(f"Input overview path does not exist: {overview_file}")
```

Comment:
Slightly more explicit (prevents giving a dir instead of file by mistake)
```suggestion
    if not hmm_path.is_file():
        raise FileNotFoundError(f"Input hmm path does not exist: {hmm_file}")

    if not overview_path.is_file():
        raise FileNotFoundError(f"Input overview path does not exist: {overview_file}")
```